File: app/src/main/java/com/tokyonth/installer/apk/CommanderCallback.java
package com.tokyonth.installer.apk;

import android.net.Uri;

import com.tokyonth.installer.bean.ApkInfoBean;

public interface CommanderCallback {

    void onStartParseApk(Uri uri);

    void onApkParsed(ApkInfoBean apkInfo);

    void onApkPreInstall(ApkInfoBean apkInfo);

    void onApkInstalled(ApkInfoBean apkInfo, int resultCode);

    void onInstallLog(ApkInfoBean apkInfo, String logText);

}

File: app/src/main/java/com/tokyonth/installer/install/CommanderCallback.kt
package com.tokyonth.installer.install

import android.net.Uri

import com.tokyonth.installer.bean.ApkInfoBean

interface CommanderCallback {

    fun onStartParseApk(uri: Uri)

    fun onApkParsed(apkInfo: ApkInfoBean)

    fun onApkPreInstall(apkInfo: ApkInfoBean)

    fun onApkInstalled(apkInfo: ApkInfoBean, resultCode: Int)

    fun onInstallLog(apkInfo: ApkInfoBean, logText: String)

}
